site/docs/data.md

```diff
@@ -96,3 +96,18 @@ Load a tab-separated values (TSV) file. This format type does not support any ad
 Load a JavaScript Object Notation (JSON) file using the TopoJSON format. The input file must contain valid TopoJSON data. The TopoJSON input is then converted into a GeoJSON format. There are two mutually exclusive properties that can be used to specify the conversion process:
 
 {% include table.html props="feature,mesh" source="TopoDataFormat" %}
+
+## Datasets
+
+Vega-Lite supports a top-level `datasets` property. This can be useful when the same data should be inlined in different places in the spec. Instead of setting values inline, specify datasets at the top level and then refer to the [named](#named) datasource in the rest of the spec. `datasets` is a mapping from name to an [inline](#inline) dataset.
+
+{: .suppress-error}
+```json
+```
+    "datasets": {
+      "somedata": [1,2,3]
```

Comment:
@ellisonbg  told me the other day that inline data has worse performance in Voyager compared to url data.  So I wonder if this `datasets` thing should offer a support for url too.  

That said, I think having inline data support to begin with is already good enough for this PR.

Reply:
(I'm just raising this point so we can consider if we should extend this.)

Reply:
That may be a bug in Voyager. I had a similar problem when I wrote an app last summer because I made deep copies of the spec. One issue was that Vega-Lite itself made copies of the data. I fixed that issue mostly but this datastore should make things actually faster. I'd try to use it in Voyager and see whether it makes it faster.